dom/nodes/Node-appendChild-text-in-script.html

```diff
@@ -10,15 +10,19 @@
 <div id="log"></div>
 <script id="script"></script>
 <script>
-var script = document.getElementById("script");
-var scriptRan = false;
-
-var df = document.createDocumentFragment();
-df.appendChild(new Text(";"));
-df.appendChild(new Text("scriptRan = true;"));
-
-assert_false(scriptRan, "script ran");
-script.appendChild(df);
-assert_true(scriptRan, "script did not run completely");
-done();
+const happened = [];
+test(() => {
+  const script = document.getElementById("script");
+  const df = document.createDocumentFragment();
+  df.appendChild(new Text("happened.push('t1');"));
+  df.appendChild(new Text("happened.push('t2');"));
+  assert_array_equals(happened, []);
+  script.appendChild(df);
+  assert_array_equals(happened, ["t1", "t2"]);
+  // At this point it's already executed so further motifications are a no-op
```

Comment:
"motifications" shuld be "notifications". 

These tests do apply to `<script type="module">`, where the exported value will always be the same no matter how many different scipts import the value, correct?